### ISSUE TITLE
Download notification for TS files

### ISSUE THREAD

**Issue by XoltErik** [subhra74/xdm#948]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

## Question

Hello! I just downloaded the new 8.0.14 and the Chrome extension on my new PC and the tool works perfectly except for one single issue I noticed. While seeing a .ts video, the download notification doesn't pop up (unlike it perfcetly did on the older version I had on Firefox). Is this a normal change due to the update or am I missing something?
Sorry for the question (probably in the wrong place?), first time asking something on GitHub!

**Comment by Damilarondo** [comment]
Well you have to go to the browser monitor settings scroll down you'll find file types and video types add a comma and TS after the last file type for both video and file types

**Comment by soorajsprakash** [comment]
@XoltErik Do the steps as @Damilarondo suggested and pls close the issue is this issue is fixed.

**Comment by XoltErik** [comment]
> Well you have to go to the browser monitor settings scroll down you'll find file types and video types add a comma and TS after the last file type for both video and file types

Yeah, I have already included TS in the lists. That's why I opened an issue
